doc/release-notes-19202.md

```diff
@@ -0,0 +1,4 @@
+## Updated settings
+
+- The `-debug=db` logging category, which was deprecated in 0.20 and replaced by
+  `-debug=walletdb` to distinguish it from `coindb`, has been removed. (#19202)
```

Comment:
I wouldn't mind if this snippet was put straight into the release notes, as there shouldn't be any conflicts with this one. This would avoid having someone to touch it again to merge it in. Though, no big deal either way.

Reply:
Done